docs/docs-content/clusters/edge/local-ui/cluster-management/link-hosts.md

```diff
@@ -0,0 +1,157 @@
+---
+sidebar_label: "Link Hosts"
+title: "Link Hosts"
+description: "Instructions for linking hosts to prepare for multi-node cluster creation."
+hide_table_of_contents: false
+sidebar_position: 30
+tags: ["edge"]
+---
+
+To create a multi-node cluster with hosts provisioned in `airgap` installation mode, the hosts must first be able
+identify and securely communicate with each other. By default, hosts that are provisioned in `airgap` installation mode
+are not aware of each other even if they are on the same network, and they do not have the credentials to communicate
+with each other securely.
+
+Host linking provides the hosts with the necessary network and security infrastructure to form a cluster together. In a
+group of linked hosts, hosts can broadcast information to all its peers. Every group has a leader node, which provides
+tokens with its IP and a One-Time Password (OTP) credentials encrypted that you can use to link other nodes.
+
+Linked hosts will sync images for the Palette agent, provider images, cluster definitions, as well as host status with
+each other. In a group of linked hosts that have not formed a cluster, only the leader node has access to functionality
+such as change host settings, upload content, and create clusters. Once a cluster is created, only the control plane
+nodes have access to features such as cluster management and can change host settings.
+
+## Link Hosts
+
+### Limitations
+
+- All hosts must be deployed in the same deployment mode. For more information, refer to
+  [Deployment Modes](../../../../deployment-modes/deployment-modes.md).
+
+- For hosts that are deployed in agent mode, all hosts must share the same Operating System (OS).
+
+### Prerequisites
+
+- Two or more hosts deployed in the same deployment mode on the same network. For more information, refer to
+  [Appliance Mode Installation](../../site-deployment/stage.md) or
+  [Agent Mode Installation](../../../../deployment-modes/agent-mode/install-agent-host.md).
+
+- All hosts must be idle. They cannot have any current cluster workloads, or are linked to another node. Refer to
+  [Delete a Cluster](./delete-cluster.md) and [Unlink Hosts](#unlink-hosts) to learn how to delete a cluster and unlink
+  a host to free it up for linking.
+
+### Procedure
+
+1. Decide on a node that you plan to use as the leader of the group. Log in to
+   [Local UI](../host-management/access-console.md) of that node. The leader node may or may not already be in a
```

Comment:
"the leader node may or may not already be in a cluster" - @lennessyy , does this mean linking works pre-cluster creation as well as post-cluster creation from leader ?
Should we highlight the day1 case instead when no hosts are linked ?

Reply:
Hmm, i think i saw this in an epic related to multi-node, but i cannot find it anymore. Can you confirm where you can still link a follower node to a leader node that is already in a cluster?